packages/gatsby-core-utils/package.json

```diff
@@ -36,9 +48,12 @@
     "file-type": "^16.5.3",
     "fs-extra": "^10.0.0",
     "got": "^11.8.3",
+    "import-from": "^4.0.0",
     "lock": "^1.1.0",
+    "lmdb": "^2.1.7",
```

Comment:
Do multiple versions of lmdb in same project cause problems? I know that for now we'll fallback to this version since `lmdb` doesn't exist in `gatsby`